db-checkbook/build_scripts/01_dataloading.py

```diff
@@ -1,16 +1,40 @@
-from dcpy.connectors.s3 import client
+from dcpy.connectors import s3
+import json
+from functools import wraps
+from pathlib import Path
+
+import pandas as pd
+import yaml
+import geopandas as gpd
+
+
+import os
+import boto3
+
+
+
+def read_s3(digital_ocean_filepath, save_file_path):
+
+    #digital_ocean_filepath example "datasets/bpl_libraries/20210208/bpl_libraries.csv"
+
+    s3_client = s3.client()
+    file = s3_client.download_file('edm-recipes', digital_ocean_filepath, save_file_path)
+    return file
```

Comment:
Looks like this function is more what you're going to use than the read_from_csv? At least in terms of moving a file from s3 to local storage.

Since our recipe file paths are pretty consistent, maybe take dataset name, extension, and version all as arguments similar to my suggestion for `read_from_csv`? And again, version could be optional and default to "latest". And at that point, you could construct the DO_filepath from these arguments.

This could also use a more descriptive name - it's downloading a file specifically from edm-recipes, so name should reflect that.

Lastly, not sure what boto3 `download_file` actually returns ([docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3/client/download_file.html) are even unclear) - since this is more a function that has a side-effect than output, I might not return anything (@alexrichey / @damonmcc thoughts?)